src/components/Sing/SequencerRuler.vue

```diff
@@ -180,47 +195,47 @@ onUnmounted(() => {
 </script>
 
 <style scoped lang="scss">
-@use "@/styles/variables" as vars;
+@use "@/styles/v2/variables" as vars;
 @use "@/styles/colors" as colors;
 
 .sequencer-ruler {
-  background: colors.$background;
+  background: var(--scheme-color-sing-ruler-surface);
+  height: 40px;
   position: relative;
   overflow: hidden;
 }
 
-.sequencer-ruler-border-bottom {
-  position: absolute;
-  left: 0;
-  top: 0;
-  width: 100%;
-  height: 100%;
-  border-top: 1px solid colors.$sequencer-sub-divider;
-  border-bottom: 1px solid colors.$sequencer-sub-divider;
-}
-
 .sequencer-ruler-playhead {
   position: absolute;
   top: 0;
-  left: -1px;
+  left: 0;
   width: 2px;
   height: 100%;
-  background: rgba(colors.$display-rgb, 0.6);
+  background: var(--scheme-color-inverse-surface);
   pointer-events: none;
   will-change: transform;
+  z-index: vars.$z-index-sing-playhead;
 }
 
 .sequencer-ruler-measure-number {
-  fill: colors.$display;
+  font-weight: 700;
+  fill: var(--scheme-color-on-surface-variant);
 }
 
 .sequencer-ruler-measure-line {
   backface-visibility: hidden;
-  stroke: rgba(colors.$display-rgb, 0.5);
+  stroke: var(--scheme-color-sing-ruler-measure-line);
+  stroke-width: 1px;
+
+  // NOTE: 最初の小節線を非表示: 必要に応じて再表示+位置合わせする
```

Comment:
コメントがちょっとおしゃれかも！（人と違うかも）
```suggestion
  // NOTE: 最初の小節線を非表示。必要に応じて再表示・位置合わせする
```